### PR TITLE
Add closing PHP tag to Readme

### PR DESCRIPTION
**Pull request by coulterpeterson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,8 @@ catch(Exception $e)
 	\QuantumPHP::add('test','warning',$e);
 }
 QuantumPHP::send();
+
+?>
```
Comment (atefBB) [inline README.md:96]: In a `php` file there's no need for the closing tag ! See https://www.php-fig.org/psr/psr-2/#22-files.

**Review by atefBB — COMMENTED** [review]
About `php` closing tag.

**Comment by coulterpeterson** [comment]
Ha forgot about this comment, and not sure why I was so insistent on form.
Disregard.

**Comment by atefBB** [comment]
^______^
close it?!

**Pull request closed without merging**